doc/whats_new.rst

```diff
@@ -22,6 +22,8 @@ BUG
 
     - Fix computation of AR coefficients across channels in :func:`mne.time_frequency.fit_iir_model_raw` by `Eric Larson`_
 
+    - Fixed maxfilter channel names extra space bug :func:``maxwell.py`` and func:``test_maxwell.py`` are updated by `Sheraz Khan`_
```

Comment:
Also, this is a Sphinx (auto doc build) link:
```
`Sheraz Khan`_
```
You need to give it some URL to link to. See the bottom of the file for examples, and add your name down there.

If you are curious about how this looks, the easiest thing is to look at the CircleCI output. Go to the circle site using the link in the CI list at the bottom of the PR, log in to CircleCI using GitHub (or other method), click "Artifacts", and navigate to the `mne-python/doc/_build/html/dev/whats_new.html` and see:

https://2757-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/whats_new.html

You can see the three errors related to bad links.